### PR TITLE
Version 5.0.0

### PR DESCRIPTION
**Pull request by wp99cp**
## Version 5 - Drawing Paths directly within the Web App

- Adds basic path drawing functionality to the web app (experimental) (5.0.0)
- Fixes a bug where the generated Excel file was buggy (5.0.0)
- Add 1:5'000 scale to the PDF export (5.0.0)
- Dependency updates

### CODE REVIEW

**Comment by wp99cp** [comment]
I decided to release this version as version 5.0.0 as after merging we need to modify the deployment of the app as well to encounter for the newly added routing server container. Thus it is not just a feature upgrade but introduces some braking changes...

File: backend/app.py [hunk 1]
```diff
@@ -154,6 +153,10 @@ def create_walk_time_table():
             )
             selected_way_points = douglas_peucker_transformer.transform(path)
 
+            # we don't use the auto waypoints if the user has disabled them
+            if not options["auto_waypoints"]:
```
Comment (maede97) [inline backend/app.py:157]: Code makes sense. However, if auto_waypoints is false, we should not compute the douglas-peucker transform (to save some time). So I guess it makes sense to move the other code in the if/else statement.

File: frontend/src/app/pages/export-settings/export-settings.component.html [hunk 2]
```diff
@@ -96,7 +113,7 @@ <h2>Marschzeittabellen erstellen</h2>
 
           <div class="slider">
             <label>Kartenmassstab: 1:{{ slider.value }}</label>
-            <mat-slider #scale min="10000" max="100000" step="5000">
+            <mat-slider #scale min="5000" max="100000" step="5000">
```
Comment (maede97) [inline frontend/src/app/pages/export-settings/export-settings.component.html:116]: können wir da einen default setzen? (z.b. 10'000 oder sogar 15'000).

**Review by maede97 — CHANGES_REQUESTED** [review]
Works on my machine. I added two comments.

File: frontend/src/app/pages/export-settings/export-settings.component.html (hunk 2 above)
Comment (wp99cp) [reply frontend/src/app/pages/export-settings/export-settings.component.html:116]: default is set here: https://github.com/cevi/automatic_walk-time_tables/blob/52290cbdde6f7c073987258c6e07a1829541e8c4/frontend/src/app/pages/export-settings/export-settings.component.ts#L32

I changed the default to 1:15'000

File: backend/app.py (hunk 1 above)
Comment (wp99cp) [reply backend/app.py]: my fix does not work... see failed run

**Review by maede97 — DISMISSED** [review]
Thank you, I guess we can merge although the two test cases failed (it seems to be only a timing issue)

**Comment by maede97** [comment]
The failing two cases are because the two routes generate > 10 PDF pages in the route. Therefore we limit to 10, but this still takes too long to generate, therefore the timeout hits.
What should we do? We can increase scaling for cypress runs? e.g. for cypress, we use 1:25000 as standard?

**Pull request merged**